components/Filters.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="filter-wrapper">
+    <span>Filter by:</span>
+    <select v-model="selectedFilter" :change="changeFilterType(selectedFilter)">
+      <option value="date">date</option>
+      <option value="name">name</option>
+      <option value="size">size</option>
+    </select>
+  </div>
+</template>
+
+<script>
+  import { mapMutations } from "vuex";
+
+  export default {
+    data: () => {
+      return {
+        selectedFilter: 'date'
+      }
+    },
+    methods: {
+      ...mapMutations({
+        changeFilterType: "changeFilterType"
```

Comment:
it should be "action" to update grid limit in store also
read the below comments ()